src/CSSList/CSSList.php

```diff
@@ -364,10 +363,10 @@ public function setContents(array $contents): void
     /**
      * Removes a declaration block from the CSS list if it matches all given selectors.
      *
-     * @param DeclarationBlock|array<array-key, Selector>|string $selectors the selectors to match
+     * @param DeclarationBlock|array<Selector>|string $selectors the selectors to match
```

Comment:
I guess `array-key` is redundant so can be removed, but also wonder if the middle type could be a `list`...